corehq/apps/userreports/tasks.py

```diff
@@ -204,37 +205,31 @@ def _run_report(backend_to_use):
     return objects
 
 
-@periodic_task(
-    run_every=crontab(minute="*/5"),
-    queue=settings.CELERY_PERIODIC_QUEUE,
-)
-def queue_async_indicators():
-    start = datetime.utcnow()
-    cutoff = start + ASYNC_INDICATOR_QUEUE_TIME
-    time_for_crit_section = ASYNC_INDICATOR_QUEUE_TIME.seconds - 10
+@periodic_task(run_every=crontab(minute="*/5"), queue=settings.CELERY_PERIODIC_QUEUE)
+def run_queue_async_indicators_task():
+    queue_async_indicators.delay()
```

Comment:
yeah when we use serial_task with max_retries=0, then even if a lot of tasks get queued up, they just stop running as soon as they start when they see that another of these tasks is currently running.